app/Console/Commands/Migrations/MigrateProjects.php

```diff
@@ -328,6 +328,7 @@ private function getConsumerFromUitId(
         $consumerSecret = $data['consumerSecret'];
 
         return new UiTiDv1Consumer(
+            Uuid::uuid4(),
```

Comment:
I think this should have been caught by PHPStan, maybe we need a higher PHPStan level? This can be done in a new ticket.